### ISSUE TITLE
How does the moneyless manifesto refute the purpose of money as a universal exchange of value?

### ISSUE THREAD

**Issue by nicksellen** [nicksellen/ponderings#2]
Fisherman sells fish for money and uses money to buy bread at some other point in time. Seems a useful thing to me. What is wrong with this?

http://www.moneylessmanifesto.org/book/the-money-delusion/real-community-requires-interdependency/ looks like the key page:

> In Debt: The First 5,000 Years, anthropologist David Graeber makes a further important point when he notes that by paying off your debts immediately (which is what money, in its more exact forms, allows us to do) we can perceive ourselves as having no further obligations to the other person.
> 
> Squaring accounts means that the two parties have the ability to walk away from each other.
> 
> By asking for a debt to be paid, and then paying it, the relationship can be considered over. 

Doesn't this just enable us to exchange goods and services with a wider range of people? We can't have personal obligations to _everyone_ right? Or is there a compelling reason we should limit our interactions to a small community?

**Comment by nicksellen** [comment]
...and noting that "social debt" very much still exists alongside money

**Comment by pauldavisthefirst** [comment]
I read the first 7 or 8 sections looking for some handling of money as a decoupler (and long-haul coupler). I couldn't find any acknowledgement of it at all. Just a lot of stuff about how a society based on personal relationships is so much better. 

I also think it misreads Graeber in its assessment that paying of a debt means that "the relationship can be considered over". Paying off a debt ends one relationship, and starts a new one, between the same two people.

**Comment by nicksellen** [comment]
What is the new relationship that starts between the same two people?

**Comment by nicksellen** [comment]
> a society based on personal relationships is so much better

If I had to _choose_ between that and the non-personal alternative, I'd choose personal relationships any day. Can we not have _both_ though?

**Comment by pauldavisthefirst** [comment]
before: you lent me something, i owe you something
after: you lent me something, i gave you what we agreed

if you don't think there's a difference, consider what happens if the debt isn't repaid:

before: you lent me something, i owe you something
after: you lent me something, i did NOT give you what we agreed

clearly different "after" cases, and i'd argue that there's a similarly important difference within each before/after pair.

**Comment by pauldavisthefirst** [comment]
Obviously, I think we can have both. But that world isn't "moneyless", right? It uses money as a tool when it is appropriate (like when I want ice cream but my freezer is broken, and i have no milk and none of my friends make ice cream) and not when there are better options (like when i need to reroof my garage and my brother is around). Although even in the latter case, money plays a role because there are other transactions that have to take place before the one(s) based on personal relationships can begin. Like buying roofing materials (which I don't have and none of my friends or neighbours makes).

**Comment by nicksellen** [comment]
I didn't get the before/after stuff :s
1. You lend me money.
2. _relationship 1 begins_
3. Time passes.
4. I give you the money back.
5. _relationship 1 ends_
6. _relationship 2 begins_? (what is it?)

**Comment by pauldavisthefirst** [comment]
It is a different relationship, but between the same two people. 

When you've lent money to someone and been repaid, your relationship with them is different than it was. It affects the likelihood that it will happen again, it affects the way you both see each other, it alters the kinds of relationships you might form with the other person's acquaintances and friends. And ditto when you're the repayer. Whether this sort of relationship is a good one seems up for discussion, but it a natural consequence of personalized lending, and quite powerful at times, I think.

**Comment by nicksellen** [comment]
> To produce one laptop you need to sell millions of its model to justify the investment in R&D and initial infrastructure alone. You need the global infrastructure that we know is polluting and destroying the ecological systems that support life on Earth. What’s more, you need technical experts and factory line workers that only do that one task for a living, dangerously limiting, as we’ve seen in chapter one, their own connection with Nature, resulting in even more social and ecological consequences.

(from http://www.moneylessmanifesto.org/book/the-moneyless-menu/the-100-local-economy/)

It suggests that if we do things globally there are unavoidable impacts _because_ they are global and mass scale. This sounds reasonable to me. What moderating factors do we have to prevent it getting out of hand? (I would lean in the direction that it is currently "out of hand" or close to it at least).

**Comment by nicksellen** [comment]
Ah, ok I get it now. The trading/debt relationship ended, but it didn't mean they severed all contact!

**Comment by serapath** [comment]
> I didn't get the before/after stuff :s
> 
> You lend me money.
> relationship 1 begins
> Time passes.
> I give you the money back.
> relationship 1 ends
> relationship 2 begins? (what is it?)

Exactly. If you are not formal enough, endless words will follow and misunderstandings will happen all the time. If you want to avoid that kind of "drifting away" you need to very carefully write down the order in which things happen and record them, so that i can answer. The language to do that is the language of financial accounting, thus bookkeeping. The basic rules are few, the application manifold

**Comment by nicksellen** [comment]
> The language to do that is the language of financial accounting, thus bookkeeping

Can you explain the social bonds between people in the language of bookkeeping? (it doesn't seem designed or suited to it to me)

**Comment by nicksellen** [comment]
But I agree about a certain formality. It's tricky to be formal about an unspecified scope of learning and understanding, but I'm hoping over time my repo will have more order emerge, it already has money as just one page in it now. It is meant to be a representation of my brain though I think, not of knowledge itself.

**Comment by serapath** [comment]
``` js
// INITIALIZE ECONOMY
var economy       = [];
var emptyBalance  = function () {
  return {
    _: {
      nick : { claims: {}},
      alex : { liabilities: {}}
    },
    journal: [ ]
  };
}
var currentTime    = economy.push(emptyBalance());
var currentJournal = economy[currentTime-1].journal;
var currentPeriod  = JSON.parse(JSON.stringify(economy[currentTime]);
var transaction;

// TRANSACTIONS
transaction = 'currentPeriod.nick['claims']['alex'] = currentPeriod.alex['liabilities']['nick'] = 1000';
currentPeriod.journal.push(transaction);
eval(transaction);
// END OF PERIOD
currentPeriod = economy[currentTime += 1];
currentPeriod.journal = [];
```

This code is buggy and should be refined, but you see what it's aiming for.

**Comment by pauldavisthefirst** [comment]
serapath, I think you really should read some Schumacher at least. He wasn't an accountant but he certainly understood enough about economics. His writing, and that of the "new left economics" groups that emerged in their wake, is precisely about how your extremely narrow view of what an economy is, how it works, what it can do, or be, and who it works for is a fundamental mistake. If you can't handle Schumacher, because he is too "out of date", then at least consider reading some of Doug Henwood's stuff at Left Business Observer, which both acknowledges "traditional" economics while demanding a more wholistic view of the interactions between economic relationships and society (and people) as a whole. He is a much better read when he's being more "technical".

**Comment by serapath** [comment]
I do not have a narrow view, i just express it with more precisicion. Math is a language. Programming Languages too. If you want to express something with as little ambiguity as possible, you have to come up with a domain specific language, otherwise you will waste a lot of words and most of the discussion will be based on different understanding about what is talked about. I made that experience in endless months, even years of discussing "the money system".

**Comment by pauldavisthefirst** [comment]
Math is a language, but not a language you can use to talk about food with. Programming languages are languages too, but pretty useless if you want to discuss the weather. 

So the question is whether or not the language that you're suggesting is required is really appropriate for the question(s) under discussion. 

The domain language of banking, or accounting, or macroeconomics, is inappropriate for discussing the impact of money on human relationships, on human perceptions of the non-human world, on non-monetary human interactions that may still be "economic". So if these are the questions under discussion, then an understanding of traditional banking, accounting and economics may be relevant but may also be substantively insufficient, or even wrong.

**Comment by nicksellen** [comment]
@serapath I like the idea of writing a program that can model the banking system, how many lines of code do you think it would be? (approx) Do you know of any existing code like this?

@pauldavisthefirst the "new left economics" left sounds like a useful avenue, have you got any recommended links or resources, my quick google search finds some paywalled papers, and some random crap...

Addressing the issue title, I think the manifesto refutes it by claiming:
- money allows international trade
- international trade requires global infrastructure
- global infrastructure damages the environment
- it also requires specialisation of labour
- specialists become disconnected with nature and so more likely to damage it

Does global infrastructure damage the environment? If so, does it _have_ to?
Are specialists disconnected from nature? If so, do they _have_ to be?

**Comment by serapath** [comment]
Not so many lines of code.
For now i would like to come up with a canonical way of how you would write down certain economic scenarios in a series of  "transactions" in the form of something like:
`currentPeriod.nick['claims']['alex'] = currentPeriod.alex['liabilities']['nick'] = 1000'`
Of course, certain "business processes" that consists of a series of "transactions" can be abstracted away using functions.

For now i would use that to (after "bootstrapping" the environment) use the javascript terminal to hack in "transactions" line by line and then see how balance sheets chance.

We can then think about dashboards to visualize the overall changes and brainstorm about what they mean :-)

**Comment by nicksellen** [comment]
Modelling bank system in software issue --> https://github.com/nicksellen/ponderings/issues/6

**Comment by pauldavisthefirst** [comment]
currentPeriod.nick['claims']['alex'] = currentPeriod.alex['liabilities']['nick'] = 1000'

tcc: line 1: type error: assumes claims[x] is numerical,measured,describable
tcc: line 1: type error: assumes liable[x] is numerical,measured,describable
tcc: line 1: arithmetic error: operator= has ambiguous semantics
tcc: line 1: object reference error: 'alex' is partially defined
tcc: line 1: object reference error: 'nick' is partially defined
tcc: line 1: temporal out of bounds: 'currentPeriod' is out of scope
tcc: stop.

**Comment by serapath** [comment]
i'll continue it in #6 then :-)

**Comment by nicksellen** [comment]
@pauldavisthefirst what is tcc? https://encrypted.google.com/search?q=tcc%20javascript didn't help me

**Comment by pauldavisthefirst** [comment]
tcc is the transactional compiler. i invented it specifically for this post. 

the latest version generates a couple of additional errors:

tcc: line 1: 'claims' is multiply-defined in other objects, and definitions are not identical
tcc: line 1: 'liabilities' is multiply-defined in other objects, and definitions are not identical

**Comment by nicksellen** [comment]
ok I get it :) - off to https://github.com/nicksellen/ponderings/issues/6 with tcc...

**Comment by nicksellen** [comment]
I obviously didn't actually read it's output carefully enough first time round :)
